### PR TITLE
remove scalafiddle.io reference

### PR DESCRIPTION
**Pull request by slee500**
https://downinspector.com/check/scalafiddle.io 

Looks like scalafiddle.io has been down for a long time. So removing references to it.

### CODE REVIEW

**Review by sjrd — APPROVED** [review]
Unfortunately, that's probably what we have to do, indeed.

File: _tour/basics.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ In this page, we will cover the basics of Scala.
 
 You can run Scala in your browser with _ScalaFiddle_. This is an easy, zero-setup way to experiment with pieces of Scala code:
```
Comment (julienrf) [inline _tour/basics.md:17]:
```suggestion
You can run Scala in your browser with _Scastie_. This is an easy, zero-setup way to experiment with pieces of Scala code:
```

Comment (julienrf) [reply _tour/basics.md:17]: We should also update the text in the other languages.

**Review by julienrf — CHANGES_REQUESTED** [review]
Thank you! Could you please also update the references to “ScalaFiddle” in the text?

File: _tour/basics.md (hunk 1 above)
Comment (slee500) [reply _tour/basics.md]: Thanks, updated. I guess once merged we can close this issue https://github.com/scala/docs.scala-lang/issues/2417 too

**Comment by slee500** [comment]
I guess once merged we can close this issue https://github.com/scala/docs.scala-lang/issues/2417 too

**Review by julienrf — APPROVED** [review]
Thank you!

**Pull request merged**